workers/unity/Assets/Gdk/Core/Worker/WorkerBase.cs

```diff
@@ -61,6 +61,18 @@ public bool Connect(ConnectionConfig config)
             return true;
         }
 
+        public void Disconnect()
+        {
+            if (Connection == null)
+            {
+                return;
+            }
+
+            View.Disconnect("WorkerBase.Disconnect() called.");
```

Comment:
Is this a meaningful disconnection reason for users? Can it be made less specific? The hardcoded `WorkerBase.Disconnect()` is a potential source of problems if this is refactored/renamed at some point.

Reply:
Yup, changed :)